src/main/java/emissary/transform/decode/JsonEscape.java

```diff
@@ -7,7 +7,7 @@
 
 public class JsonEscape {
 
-    private final static String ESCAPES = "ntr\"'/";
+    private final static String ESCAPES = "ntr\"'/\\";
```

Comment:
The ordering should be `private static final`

Reply:
this has been updated.  Sorry for the late response, I just realized my email address for notifications was old, it is now current